srv/components/provisioner/scripts/cortx_prep.sh

```diff
@@ -0,0 +1,291 @@
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+set -euE
+export LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/node_prep.log}"
+mkdir -p $(dirname "${LOG_FILE}")
+
+PRVSNR_ROOT="/opt/seagate/cortx/provisioner"
+localhost_ip="127.0.0.1"
+minion_id="srvnode-0"
+user_name=
+uid=
+group=
+gid=
+repo_url=
+ISO_CORTX_PATH="/opt/isos/cortx-*-single.iso"
+ISO_OS_PATH="/opt/isos/cortx-os-*.iso"
+nodejs_tar="http://cortx-storage.colo.seagate.com/releases/cortx/github/integration-custom-ci/centos-7.8.2003/custom-build-1969/3rd_party/commons/node/node-v12.13.0-linux-x64.tar.xz"
+
+usage()
+{
+    echo "
+Usage: Usage: $0 [options]
+
+Configure Cortx prerequisites locally.
+
+General options:
+
+Options:
+  -t|--target-build BUILD_URL        Target Cortx build to deploy
+"
+}
+
+parse_args()
+{
+    echo "parse_args(): parsing input arguments" >> $LOG_FILE
+
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+            -t|--target-build)
+                if [[ -z "$2" ]]; then
+                    echo "Error: URL for target cortx build not provided" && usage && exit 1;
+                fi
+                repo_url=$2
+                shift 2
+                ;;
+            -h|--help)
+                usage; exit 0
+            *) echo "Invalid option $1"; usage; exit 1;;
+        esac
+    done
+}
+
+config_local_salt()
+{
+
+    # 1. Point minion to local host master in /etc/salt/minion
+    # 2. Restart salt-minion service
+    # 3. Check if 'salt srvnode-0 test.ping' works
+
+    if ! rpm -qa | grep -iEwq "salt|salt-master|salt-minion|cortx-prvsnr"; then
+        echo "ERROR: salt packages are not installed, please install salt packages and try again." | tee -a ${LOG_FILE}
+        exit 1
+    fi
+
+    minion_file="${PRVSNR_ROOT}/srv/components/provisioner/salt_minion/files/minion_factory"
+    master_file="${PRVSNR_ROOT}/srv/components/provisioner/salt_master/files/master"
+
+    yes | cp -f "${master_file}" /etc/salt/master
+    yes | cp -f "${minion_file}" /etc/salt/minion
+
+    #local line_to_replace=$(grep -m1 -noP "master: " /etc/salt/minion|tail -1|cut -d: -f1)
+    #echo "Setting minion to local master in /etc/salt/minon" >> ${LOG_FILE}
+    #yes | cp -f /etc/salt/minion /etc/salt/minion.bkp
+    #sed -i "${line_to_replace}s|^master:.*|master: ${localhost_ip}|" /etc/salt/minion
+    echo "Done" | tee -a ${LOG_FILE}
+
+    echo $minion_id > /etc/salt/minion_id
+
+    echo "Restarting the required services" | tee -a ${LOG_FILE}
+    systemctl start salt-master
+    systemctl restart salt-minion
+    echo "Done" | tee -a ${LOG_FILE}
+
+    echo "INFO: Waiting for key of $minion_id to become connected to salt-master" >> ${LOG_FILE}
+    try=1; max_tries=10
+    until salt-key --list-all | grep "srvnode-0" >/dev/null 2>&1
+    do
+        if [[ "$try" -gt "$max_tries" ]]; then
+            echo "ERROR: Key for salt-minion $minion_id not listed after $max_tries attemps." >> ${LOG_FILE}
```

Comment:
Remove keyword `salt-minion`

Reply:
It's debug log, redirected to LOG_FILE directly, added 'DEBUG' keyword now